src/blockchains/eth_generator/consts.test.ts

```diff
@@ -0,0 +1,14 @@
+import "jest-extended"
+import {evaluator, IWideDataType, PUSH_CONSTRUCTION} from "./consts";
+
+describe("Regression test for empty value or decimals", () => {
```

Comment:
`skip()` this tests for now

Reply:
The test checks, that independent constructor module uses default value as fallback and if provided uses the provided initial value. There could be another test on ducor-backend level, that takes the whole system behaviour, not just eth-generator.